### DISCUSSION TITLE
LTI 1.3 with GitHub Classroom

### DISCUSSION THREAD

**Discussion by amanda-nash** [community/community#17757] [GitHub Education]
I’m looking for information about whether GitHub Classroom will support [LTI 1.3](https://www.imsglobal.org/activity/learning-tools-interoperability). Is there any news about updates? I don’t see any on the [connect LMS to GitHub Classroom](https://docs.github.com/en/education/manage-coursework-with-github-classroom/teach-with-github-classroom/connect-a-learning-management-system-to-github-classroom) page.

**Comment by samus-aran** [comment]
Hi @amanda-nash 👋 The team are currently working on this feature so do stay tuned. If you're a verified teacher (and joined [Global Campus](https://education.github.com/teachers)) we are looking for [teachers to preview this](https://github.com/community/Global-Campus-Teachers/discussions/131) to help us launch.
